postgresStore.go

```diff
@@ -48,30 +45,30 @@ func (p *PostgresStore) Create(task Task) (int, error) {
 	return taskid, nil
 }
 
-func (p *PostgresStore) NewTaskSession(task Task) error {
+func (d *DBStore) NewTaskSession(task Task) error {
 	username := "app"
```

Comment:
We don't actually need `username` at this point, do we? So it would probably be wise to either eliminate usernames altogether, or go ahead and add the functionality to associate tasks with different named users. We don't want to get stuck sort of halfway towards that.